src/analyses/README.md

```diff
@@ -18,7 +18,16 @@ This is obsolete.
 
 \subsection analyses-flow-insensitive-analysis Flow-insensitive analysis (flow_insensitive_analysist)
 
-To be documented.
+Framework for flow-insensitive analyses. Maintains a single global domain which
+instructions are invited to transform in turn. Unsound (terminates too early) if
+(a) there are two or more instructions that incrementally reach a fixed point,
+for example by walking a chain of pointers and updating a points-to set, but
+(b) those instructions are separated by instructions that don't update the
+domain (for example, SKIP instructions) unless the domain carefully avoids this
+fate. Therefore, not recommended for new code.
+
+Only current user in-tree is \ref value_set_analysis_fit and its close
```

Comment:
Is `flow_insensitive_analysist` still active, or has it been/will it be deprecated like `static_analysist` ? If so should we clearly state that here? (there is the ambiguous "Therefore, not recommended for new code.", but perhaps it should be more forceful)

Reply:
AFAICT it is not explicitly deprecated, that's just my recommendation